src/utils/url.ts

```diff
@@ -1,13 +1,15 @@
-export const getCurrentPath = () => {
+const getLocalePostfix = (locale: string) => (locale !== 'en' ? `/${locale}` : '');
+
+export const getCurrentPath = (locale: string) => {
   if (typeof window !== 'undefined') {
-    return window.location.href;
+    return `${window.location.href}${getLocalePostfix(locale)}`;
```

Comment:
https://quran-com-git-link-copying-should-include-locale-qurancom.vercel.app/id/reciters/1/2
Not sure if this is a corner case or not

But if we click "Salin Tautan" / "Copy".

The copied URL should be not be changed `https://quran-com-git-link-copying-should-include-locale-qurancom.vercel.app/id/reciters/1/2/id`. It should stay the same

Reply:
Good catch. This is not expected. Fixing it